libs/watt/.storybook/main.ts

```diff
@@ -1,26 +1,63 @@
-import type { StorybookConfig } from '@storybook/angular';
+import path from 'path';
+import type { UserConfig } from 'vite';
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
 
-const config: StorybookConfig = {
-  staticDirs: [
-    {
-      from: '../src/assets',
-      to: 'assets/watt',
-    },
-  ],
```

Comment:
issue: this will need to be kept somehow because the custom icons as well as the visual spacing reference image (under Spacing) do not load.

Reply:
Good catch, will have a look